packages/language-flow/src/index.ts

```diff
@@ -1,15 +1,37 @@
-import { IFromQueryOptions, WrapType } from '@gql2ts/types';
-import { DEFAULT_OPTIONS as TS_OPTIONS } from '@gql2ts/language-typescript';
+import {
+  IFromQueryOptions,
+  WrapType,
+  InterfaceAndTypeBuilder,
+  TypePrinter,
+  NamespaceGenerator
+} from "@gql2ts/types";
+import { DEFAULT_OPTIONS as TS_OPTIONS } from "@gql2ts/language-typescript";
+import { pascalize } from "humps";
 
 export const FLOW_WRAP_PARTIAL: WrapType = partial => `$SHAPE<${partial}>`;
+export const FLOW_INTERFACE_NAMER: WrapType = name => `${pascalize(name)}`;
+export const FLOW_INTERFACE_BUILDER: InterfaceAndTypeBuilder = (name, body) =>
+  `export type ${name} = ${body}`;
+export const FLOW_ENUM_NAME_GENERATOR: WrapType = name => `${pascalize(name)}`;
+export const FLOW_TYPE_PRINTER: TypePrinter = (type, isNonNull) =>
```

Comment:
should we add `Type` or something to avoid conflicts with the interfaces/enums? Or just assume that there shouldn't be duplicates. Not sure what the best move is here

Reply:
I vote no to adding a suffix to avoid conflicts. GraphQL doesn't allow duplicate type names. Enums should always be in SCREMING_CASE. As for conflicts with the `IGraphQL*` types, they could be separated by moving them to a different file.

Reply:
Upon further consideration, you seem to be right about having type name conflicts. https://github.com/arahansen/gql2ts/pull/1

Reply:
@arahansen i think suffixing this with `Type` or something might be the best way forward. 

It's not the best solution, but it works for the typescript version